simtools/db_handler.py

```diff
@@ -1267,63 +1271,44 @@ def _getTaggedVersion(dbName, version="Current"):
 
         return tags["Tags"][version]["Value"]
 
-    def insertFileToDB(self, file, dbName=DB_CTA_SIMULATION_MODEL, **kwargs):
+    def insertFileToDB(self, fileName, dbName=DB_CTA_SIMULATION_MODEL, **kwargs):
         """
-        Insert a file to the DB.
+        Insert file to the DB.
 
         Parameters
         ----------
+        fileName: str or Path
+            The name of the file to insert (full path).
         dbName: str
             the name of the DB
-        file: str or Path
-            The name of the file to insert (full path).
         **kwargs (optional): keyword arguments for file creation.
             The full list of arguments can be found in, \
             https://docs.mongodb.com/manual/core/gridfs/#the-files-collection
             mostly these are unnecessary though.
+            kwargs can be used only in case of passing a single file to the function.
 
         Returns
         -------
         file_id: gridfs "_id"
-            If the file exists, returns the "_id" of that one, otherwise creates a new one.
+            If the file exist, returns its "_id", otherwise creates the file and returns its id.
```

Comment:
Both sentences are not great I think. I suggest to write

`If the file exists, return its "_id", otherwise insert the file and return its newly created DB "_id".`